File: anika.dovob/src/main/java/com/example/anika/homework/chapter6/homework3/Product.java
package com.example.anika.homework.chapter6.homework3;

import jakarta.persistence.*;

@Entity(name = "Hw3Product")
@Table(name = "product_hw3")
public class Product {

    @Id
    @GeneratedValue
    @Column(name = "product_id")
    private Long id;

    private String name;

    public Product() {
    }

    public Long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: joon.bug/src/main/kotlin/com/example/joonbug/chapter6/homework3/Product.kt
package com.example.joonbug.chapter6.homework3

import jakarta.persistence.*

/**
 * 숙제 3: 다대다 중간 엔티티 - Product 엔티티
 */
@Entity(name = "Hw3Product")
@Table(name = "hw3_product")
class Product(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,

    var name: String = "",

    @OneToMany(mappedBy = "product")
    var orders: MutableList<Order> = mutableListOf()
)